obs-browser-plugin.cpp

```diff
@@ -648,7 +685,7 @@ bool obs_module_load(void)
 #if defined(__APPLE__) && CHROME_VERSION_BUILD < 4183
 	// Make sure CEF malloc hijacking happens early in the process
 	if(isHighThanBigSur())
-		obs_browser_initialize();
+		obs_browser_initialize(nullptr, nullptr);
```

Comment:
Is there a specific reason why we shouldn't pass the data in that case?

Reply:
we don't have the data here.